Document `workspaceOpeners` field of package.json (#533)
atom/flight-manual.atom.io: content/hacking-atom/sections/package-word-count.md

* `snippets`: an Array of Strings identifying the order of the snippets your package needs to load. If not specified, snippets in the `snippets` directory are added alphabetically.
* `activationCommands`: an Object identifying commands that trigger your package's activation. The keys are CSS selectors, the values are Arrays of Strings identifying the command. The loading of your package is delayed until one of these events is triggered within the associated scope defined by the CSS selector.  If not specified, the `activate()` method of your main export will be called when your package is loaded.
* `activationHooks`: an Array of Strings identifying hooks that trigger your package's activation. The loading of your package is delayed until one of these hooks are triggered. Currently, there are two activation hooks: `language-package-name:grammar-used` (e.g., `language-javascript:grammar-used`) and `core:loaded-shell-environment`.

> * `workspaceOpeners`: An Array of Strings identifying URLs that should trigger your package's activation when opened. For example, say your package registers a custom opener for `atom://my-custom-panel`. By including that string in `workspaceOpeners`, your package will defers its activation until that URL is opened.

Reviewer: ```suggestion
* `workspaceOpeners`: An Array of Strings identifying URIs that trigger your package's activation. For example, say your package registers a custom opener for `atom://my-custom-panel`. By including that string in `workspaceOpeners`, your package will defer its activation until that URI is opened.
```
